### DISCUSSION TITLE
retrieve country, region and city from exif GPS data in python (HEIC - filetype).

### DISCUSSION THREAD

**Discussion by ttuschak** [coder/code-server#6598] [Q&A]
Can anybody help me how i can get country, region and city from images od different formats (e.g. HEIC,JPG, ARW...) with python?
i m stuck once i have the touple values for Latitude and Longitude...
What i am trying to acieve is to build a sqlite db with all relevant exif data. The end goal is to select files based on Date and Location (best whould be something like country, region and/or city) so i can rename them with date and location...
Any hints are appreciated

**Comment by code-asher** [comment]
This is not specific to code-server is it?  You will probably have more luck asking on Stack Overflow or some general programming channel somewhere.
